src/deck-options.md

```diff
@@ -426,7 +426,7 @@ answer; for that please see [this section](templates/fields.md#special-fields).
 ### FSRS
```

Comment:
Delete one hash. Then move the 'Advanced' heading to the bottom.

Reply:
Check my latest commit